src/inmanta/server/services/compilerservice.py

```diff
@@ -570,6 +570,8 @@ async def request_recompile(
         env_vars: Optional[Mapping[str, str]] = None,
         partial: bool = False,
         removed_resource_sets: Optional[List[str]] = None,
+        notify_failed_compile: Optional[bool] = None,
+        failed_compile_message: str = None,
```

Comment:
We should also document these arguments here. It should definitely describe the behavior when notify_failed_compile is set to None as this is not very intuitive.